handlers/meshery_filter_handler.go

```diff
@@ -18,6 +18,14 @@ type MesheryFilterRequestBody struct {
 	FilterData *models.MesheryFilter `json:"filter_data,omitempty"`
 }
 
+// swagger:route GET /api/experimental/filter/file/{id} FiltersAPI idGetFilterFiles
+// Handle GET request for filter file with given id
+//
+// Returns the list of all the filter files saved by the current user with the given id
```

Comment:
as far as I can guess, this will return a single filter file! we aren't attaching multiple files with same id.

Reply:
Done